vspace/src/main/java/edu/asu/diging/vspace/core/aspects/ExhibitionDataAspect.java

```diff
@@ -119,7 +119,7 @@ private Object redirectRequest(ProceedingJoinPoint jp, String spaceId, String mo
         }
         // If user is not logged in and exhibition is not active, show maintenance page.
         if(authFacade.getAuthenticatedUser()==null && !exhibitionMode.equals(ExhibitionModes.ACTIVE)) {
-            return "maintenance";
+            return "exhibition/maintenance";
```

Comment:
Noticed that the path to the maintenance file was incorrect. Should I let this fix remain as part of this PR?